### ISSUE TITLE
[3.3] Repeaters mismatch between slug and key..

### ISSUE THREAD

**Issue by bobdenotter** [bolt/bolt#6438]
There's still some mismatch between the keys and slugs for contenttypes, when it comes to repeaters:

Details
-------

 - Relevant Bolt Version: [ 3.2 | 3.3 | master ]

Reproduction
------------

Create a simple contenttype, like: 

```
foo:
    name: Bars
    slug: bars
    singular_name: Bar
    singular_slug: bar
    fields:
        title:
            type: text
            class: large
        slug:
            type: slug
            uses: [ title ]
        images:
            type: repeater
            fields:
                repeattitle:
                    type: text
                repeatimage:
                    type: image
```

Add a record. 

Edit the record.twig template: 

```
{% block main %}

    <h1>{{ record.title }}</h1>

<hr>

{% for section in record.images %}
    {{ dump(section) }}
    image: {{ section.get('repeatimage')|join(',')  }} <br>
    title: {{ section.get('repeattitle') }}

{% else %}

    <p>No repeaters.</p>

{% endfor %}

<hr>
```

In the browser it now looks like this: 

![screen shot 2017-03-01 at 17 57 44](https://cloud.githubusercontent.com/assets/1833361/23471134/98d2505a-fea8-11e6-975a-a4ba211c025d.png)

Now, edit the contenttype, to look like this: 

```
bars:  # note: foo -> bars
    name: Bars
    slug: bars
    singular_name: Bar
    singular_slug: bar
    fields:
        title:
....
```

Go back to editing the record. Add a repeater block or two, and back to the browser: 

![screen shot 2017-03-01 at 18 00 46](https://cloud.githubusercontent.com/assets/1833361/23471261/0586c17c-fea9-11e6-9d53-33554ef8969f.png)

@rossriley Is this fixable? Or should we add a warning for this, for now?

**Comment by rossriley** [comment]
it should be fixable yes, nowhere should use the slug, everywhere should use the contenttype name which I thought we'd got every instance of but it appears not.

I'll debug and get a fix in.

**Comment by bobdenotter** [comment]
Fixed in #6453
